### DISCUSSION TITLE
Adding more hooks in teamcity

### DISCUSSION THREAD

**Discussion by MikolajJaworski1996** [pascal-zarrad/tc-discord-webhooks#32] [Ideas]
Hi guys. I can't find any info if that is possible so i just write here on ideas.

With my boss We come to conclusion, that our build channel on discord is a little dirty now. Can You add option (or if there is one, can you show me how to do it) - to add 2 or 3, or even infinity webhooks to one teamcity server, so i can get notification from projects on other channels.

**Comment by pascal-zarrad** [comment]
Hi!
Unfortunaly, the only way right now to configure multiple webhooks is to use multiple users.
Every user can have his own webhook configuration and therefore an own webhook link.
Setting up multiple webhook URLs using the same user is not possible right now.

Feel free to create an issue for that. Maybe someone wants to contribute such a feature in the future.

**Reply by MikolajJaworski1996** [reply]
Ooo thats is something that will resolve my problem i did't knew i can add other webhook to other user. THX
